app/assets/stylesheets/question_index.scss

```diff
@@ -0,0 +1,25 @@
+.question-body {
+    font-size: larger;
+    font-family: -webkit-body;
+  }
+
+  .button-round {
+    margin-right: 347px;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, border-width, color, font-family, font-size, margin-right, padding-bottom, padding-left, padding-right, padding-top